highest-scoring-word/solution.js

```diff
@@ -0,0 +1,6 @@
+function high(x){
+  let testArr = x.split(' ');
+  let maxArr = testArr.map(word => word.split('').reduce((acc, curr) => acc + (curr.charCodeAt(0)-96), 0));

```

Comment:
forEach() would be more appropriate to use than .map() for this kata.
It works, but like we said yesterday, this kata's challenge was not to transform data but to evaluate data.

forEach(), like .map(), is also declarative (not iterative), even though it's iterating behind the scenes.